subprojects/packagefiles/pcre/meson.build

```diff
@@ -65,14 +65,23 @@ foreach t : [ 'long long', 'unsigned long long' ]
   cdata.set('HAVE_@0@'.format(t.underscorify().to_upper()), cc.has_type(t))
 endforeach
 
+if host_machine.system() == 'windows'
```

Comment:
I'd change this to get_argument_syntax() == 'msvc'. MinGW works fine.

Reply:
done